lua/gt.lua

```diff
@@ -79,8 +79,10 @@ return function (net_conf, lls_conf, numa_table)
 		math.max(lls_conf.mailbox_max_pkt_sub,
 		gt_conf.max_pkt_burst)
 
-	local gt_lcores = staticlib.alloc_lcores_from_same_numa(numa_table,
-		n_lcores)
+	local gt_lcores_tbl =
+		staticlib.alloc_lcores_evenly_from_all_numa_nodes(numa_table,
+			n_lcores, 0)
+	local gt_lcores = staticlib.convert_table_to_array(gt_lcores_tbl)
```

Comment:
We don't need to introduce variable `gt_lcores_tbl`, just wrap the call of `alloc_lcores_evenly_from_all_numa_nodes()` with `convert_table_to_array()`.